models/controlled-documents/src/types.ts

```diff
@@ -259,7 +259,7 @@ export class TDocument extends TDoc implements Document {
     state!: DocumentState
 
   @Prop(TypeCollaborativeDoc(), documents.string.CollaborativeDocument)
-    content!: CollaborativeDoc
+    content!: Ref<Blob> | null
```

Comment:
Minor: as an idea we may have kept a separate type for collaborative content which would simply be an alias for `Ref<Blob>`. It could improve readability and make it simpler for further refactoring if needed. WDYT?

Reply:
yeah will do, can you suggest any name? :)

Reply:
I'd go with something generic and vague like `CollaborativeContent` which will be clarified in the type definition itself.

Reply:
Done